.travis.yml

```diff
@@ -12,6 +12,9 @@ addons:
 
 install:
   - rustup target add thumbv6m-none-eabi
+  - rustup component add rustfmt
 
 script:
-  - ./scripts/build.sh
+  - |
+    cargo fmt -- --check &&
+    ./scripts/build.sh
```

Comment:
Might be better to add this check to `build.sh`, to prevent surprises when the pull request is submitted. Not sure. What do you think?

Reply:
I'm not merging until you had a chance to reply to this comment. Feel free to just merge yourself, once you've addressed this or decided not to.